src/webhooks/handleDisputeLost.ts

```diff
@@ -0,0 +1,52 @@
+import Stripe from 'stripe';
+import { UsersService } from '../services/users.service';
+import { StorageService } from '../services/storage.service';
+import CacheService from '../services/cache.service';
+import { AppConfig } from '../config';
+import handleLifetimeRefunded from './handleLifetimeRefunded';
+import { PaymentService } from '../services/payment.service';
+import { FastifyBaseLogger } from 'fastify';
+
+interface HandleDisputeLostProps {
+  charge: Stripe.Dispute;
+  stripe: Stripe;
+  paymentService: PaymentService;
+  usersService: UsersService;
+  storageService: StorageService;
+  cacheService: CacheService;
+  log: FastifyBaseLogger;
+  config: AppConfig;
+}
+
+export async function handleDisputeLost({
+  charge,
+  stripe,
+  paymentService,
+  usersService,
+  storageService,
+  cacheService,
+  log,
+  config,
+}: HandleDisputeLostProps) {
+  const isDisputeLost = charge.status === 'lost';
+  const chargeId = charge.charge as string;
+
```

Comment:
Return if the dispute is not lost, so you reduce the cognitive complexity of the if that wraps the entire `try-catch`